stablehlo/dialect/VhloOps.h

```diff
@@ -18,53 +18,13 @@ limitations under the License.
 #define STABLEHLO_DIALECT_VHLO_OPS_H
 
 #include "llvm/ADT/StringRef.h"
-#include "mlir/Dialect/Quant/QuantTypes.h"
-#include "mlir/Dialect/Shape/IR/Shape.h"
 #include "mlir/IR/Attributes.h"
-#include "mlir/IR/Builders.h"
-#include "mlir/IR/BuiltinAttributes.h"
-#include "mlir/IR/BuiltinTypes.h"
 #include "mlir/IR/Dialect.h"
 #include "mlir/IR/DialectImplementation.h"
-#include "mlir/IR/Location.h"
 #include "mlir/IR/MLIRContext.h"
-#include "mlir/IR/OpDefinition.h"
 #include "stablehlo/dialect/Version.h"
```

Comment:
We can probably clean up some or maybe even all of these includes from this file, given that they all seem to come from included files anyway?

Reply:
Removed several, with Dialect back in this file I think the list is good. Don't want to rely too much on headers including other headers for us